common/model-views.cpp

```diff
@@ -1391,6 +1391,11 @@ namespace rs2
                             }
                             catch (...) {}
                         }
+                        else
+                        {
+                            error_message = to_string() << "Resolution: " << width << " x " << height
```

Comment:
No need for spaces around the `x`, and the `:` isn't needed, either

Reply:
Done - Image updated